frontend/package.json

```diff
@@ -1,34 +1,39 @@
 {
   "name": "frontend",
-  "version": "0.1.0",
+  "version": "1.0.0",
+  "description": "Frontend part of software angels attempt",
+  "main": "index.js",
+  "author": "alramalho",
+  "license": "MIT",
   "private": true,
-  "dependencies": {
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "react-scripts": "4.0.2"
-  },
   "scripts": {
-    "start": "PORT=8080 react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "webpack serve --mode=development --host 0.0.0.0",
```

Comment:
what led you to start the project by using create-react-app and then doing (I assume) an eject?

Reply:
Firstly I've decided to go with `create-react-app` because I did not want to end up losing too much time with configurations. Although, once I integrated the library `material-table` a lot of strange conflicts were appearing. In order to better solve them I felt the need to a more granular control over what was happening. Therefore, I decided to ditch `create-react-app` in favor of `babel`+`webpack`